controller/main.go

```diff
@@ -122,13 +122,14 @@ func (c *controller) MarkSynced(l log.Logger) {
 
 func main() {
 	var (
-		port       = flag.Int("port", 7472, "HTTP listening port for Prometheus metrics")
-		config     = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
-		namespace  = flag.String("namespace", os.Getenv("METALLB_NAMESPACE"), "config / memberlist secret namespace")
-		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file (only needed when running outside of k8s)")
-		mlSecret   = flag.String("ml-secret-name", os.Getenv("METALLB_ML_SECRET_NAME"), "name of the memberlist secret to create")
-		deployName = flag.String("deployment", os.Getenv("METALLB_DEPLOYMENT"), "name of the MetalLB controller Deployment")
-		logLevel   = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", strings.Join(logging.Levels, ", ")))
+		port          = flag.Int("port", 7472, "HTTP listening port for Prometheus metrics")
+		config        = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
+		namespace     = flag.String("namespace", os.Getenv("METALLB_NAMESPACE"), "config / memberlist secret namespace")
+		kubeconfig    = flag.String("kubeconfig", "", "absolute path to the kubeconfig file (only needed when running outside of k8s)")
+		mlSecret      = flag.String("ml-secret-name", os.Getenv("METALLB_ML_SECRET_NAME"), "name of the memberlist secret to create")
+		deployName    = flag.String("deployment", os.Getenv("METALLB_DEPLOYMENT"), "name of the MetalLB controller Deployment")
+		logLevel      = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", strings.Join(logging.Levels, ", ")))
+		disableSlices = flag.Bool("disable-slices", false, "Disable the usage of endpointslices")
```

Comment:
I'd call this `disable-epslices` and mention the fact that it defaults to endpoints in the description without relying on the autodiscovery mechanism

Reply:
Also, EndpointSlices or endpoint slices in the description

Reply:
Done